images/builder/Dockerfile

```diff
@@ -2,8 +2,8 @@
 # SPDX-License-Identifier: Apache-2.0
 
 ARG COMPILERS_IMAGE=quay.io/cilium/image-compilers:6ed96e618f68cd941b62b546fc8d161db0144d84@sha256:5245610a830b14972d5b287fc0318ac6ab0d420e4afc9b5db9032a04fd706d32
-ARG CILIUM_RUNTIME_IMAGE=quay.io/cilium/cilium-runtime:d1f58566ba3f12d7805d502cf137f8b6d27ac0fd@sha256:8dd763b845816f9e42122e38dc555fb6d85142132d0e060fe651c3fe53c83c44
-ARG TESTER_IMAGE=quay.io/cilium/image-tester:eb76f0d4d585946bb0267b4c4478cceed17bbd87@sha256:c9a598cdc4843e62ab4e430f36343ca06d5292506261f17e466e0f1cbb48ddbb
+ARG CILIUM_RUNTIME_IMAGE=quay.io/cilium/cilium-runtime:9922065e5fa27684f4e5848e450afc1b5d7c7f20
+ARG TESTER_IMAGE=quay.io/cilium/image-tester:70724309b859786e0a347605e407c5261f316eb0@sha256:89cc1f577d995021387871d3dbeb771b75ab4d70073d9bcbc42e532792719781
```

Comment:
@TrevorTaoARM can you remove this line change? Thanks

Reply:
> @TrevorTaoARM can you remove this line change? Thanks
Yes, I just rebased the related Dockerfiles, and now it seems that only the 3 shell scripts are changed now. 
Did I forget any other steps to do?